app/transport/http/user.go

```diff
@@ -27,7 +27,7 @@ func (uh UserHandler) CreateUser() http.Handler {
 		}()
 
 		if err := user.Validate(); err != nil {
-			WriteJSONResponse(w, http.StatusBadRequest, Response{"Validation error: " + err.Error()}, uh.logger)
+			WriteJSONResponse(w, http.StatusBadRequest, Response{MsgValidationErr, err.Error()}, uh.logger)
```

Comment:
Usually, the main messages of the response are less specific and more generic. For example, any 400 response will have the same `Message` (like `Bad Request`) but different `Details`. Otherwise, you will end up with tons of different constants in your `const.go` file.